src/SolverAlsWrmf.cpp

```diff
@@ -147,11 +193,51 @@ T als_explicit(const dMappedCSC& Conf,
       // X_nnz = [x_bias, ..., 1]
       // Y_new should be [..., y_bias]
       if (solver == CONJUGATE_GRADIENT) {
-        Y_new = cg_solver_explicit<T>(X_nnz, confidence, init, lambda_use, cg_steps);
+        if (!with_implicit_features) {
+          if (with_biases) {
+            auto init = drop_row<T>(Y.col(i), !is_x_bias_last_row);
```

Comment:
I'm a bit afraid of using `auto` with arma objects. I had some segfaults and don't use `auto` with arma structures anymore. Better to use `const arma::Mat<T>` here.
In fact [arma docs](http://arma.sourceforge.net/faq.html) discourage using `auto`:
> Use of C++11 auto is not recommended with Armadillo objects and expressions.